lib/request.js

```diff
@@ -13,15 +13,105 @@
  * contents is a violation of applicable laws.
  */
 
-const config = require('config');
 const request = require('request');
 
-const options = {
-  baseUrl: config.host,
-  json: true,
-  timeout: config.timeout || 5000
-};
+const config = require('config');
+const { Errors, HTTPMethod } = require('./constants');
+const KinveyError = require('./kinvey-error');
+const { isConnectionError, isNullOrUndefined } = require('./utils');
+
```

Comment:
The CLI is growing quite a bit, I think it could be helpful to start adding general descriptions at the top of each file to describe its purpose (with or without JSDoc - your call)

Reply:
Done